src/Components/Web/src/Forms/InputFile/BrowserFileStream.cs

```diff
@@ -67,6 +87,39 @@ public override async ValueTask<int> ReadAsync(Memory<byte> buffer, Cancellation
             return bytesRead;
         }
 
-        protected abstract ValueTask<int> CopyFileDataIntoBuffer(long sourceOffset, Memory<byte> destination, CancellationToken cancellationToken);
+        private async Task<Stream> OpenReadStreamAsync(RemoteBrowserFileStreamOptions options, CancellationToken cancellationToken)
```

Comment:
Technically we consume a [`RemoteBrowserFileStreamOptions`](https://source.dot.net/#Microsoft.AspNetCore.Components.Web/Forms/InputFile/RemoteBrowserFileStreamOptions.cs,1dc0517a518759e5), but I'm tempted to remove it to avoid the additional complexity / increase in "concept count". We're using pretty safe defaults for regular JSDataStreams and there may not be too much use of the file stream specific options. Any objections?

Reply:
It seems like we could probably get away with it, because:

 * It's not a source or binary breaking change (if we left `RemoteBrowserFileStreamOptions` in existence but marked it as `[Obsolete]`). It's only a change of behavior in that we'd now be ignoring those options and using the defaults from JS interop.
 * Very few people will likely have tried to customize this, and even if they have, we believe our new JS interop streaming defaults should be good for everyone, right?

If this is the case then I'd support marking `RemoteBrowserFileStreamOptions` as obsolete and no longer respecting it. @javiercn @pranavkm Opinions?

Reply:
> If this is the case then I'd support marking RemoteBrowserFileStreamOptions as obsolete and no longer respecting it. @javiercn @pranavkm Opinions?

Is either of you able to access chemweb.azurwebsites.net? That might give us some sense of how frequently this option gets used and perhaps changes our opinions.

Reply:
Unfortunately not.

![image](https://user-images.githubusercontent.com/14852843/125534487-ad4eecc1-f628-4a81-a6fc-fca61be1b085.png)